### ISSUE TITLE
Home app "no response"

### ISSUE THREAD

**Issue by SheriffOfNotts** [KraigM/homebridge-nest#84]
Hi guys I'm having a problem with the home app and the thermostat. Everything else works fine - harmony, WeMo etc, but he thermostat says no reponse unless I amend temperature using another app e.g. Eve. Has anyone else seen this? Any ideas?

**Comment by leoneleone** [comment]
check your nodes version. update to the latest version of node

**Comment by SheriffOfNotts** [comment]
Hi there, thanks for the reply. I’m currently on v4.5.0? Do i need to upgrade to 6.7.0? If I do please can you let me know how to.

Regards,

Craig.

**Comment by leoneleone** [comment]
curl deb.nodesource.com/setup_6.x | sudo bash
sudo apt-get install nodejs

**Comment by M0rph3v5** [comment]
it worked shortly before I'm getting this as well. Running 4.6.0 on a raspberry pi.

**Comment by CynanX** [comment]
Hi,

I get 'no response' if my Nest is set to the lowest temperature setting of 9C.

If it's 9.5 or higher then I get a response.

Bug?

--Cynan

**Comment by jata1** [comment]
hi, i also get the issue where 'no response' or 'updating' in the home app for my nest thermostat. I have done a new install on a raspberry pi 2 or 3 days ago so everything is latest. using the v6 API. 

To fix the issue, i just open the nest app which always works fine. 

after opening nest app, i go back to home app and it is connected and working fine.

any ideas to help me fix this?

**Comment by erfinb** [comment]
Something to add to this, I get the same no response error in the Home app, unless I am on my home's wifi network. So if I'm out of the house it says "no response" and then I get to the house and connect to wifi and it works without any issue in the home app. The Nest app works all of the time. Any ideas?

**Comment by classyuser** [comment]
@erfinb Have you setup a device as your home hub?  To my understanding, Homebridge isn't a substitute for having an AppleTV or iPad as your home hub.  [See instructions from Apple for setup.](https://support.apple.com/en-us/HT207057)

**Comment by erfinb** [comment]
Yeah it's all setup with my Apple TV. I can see all other devices off network, just not the Nest through HomeKit...

**Comment by jaytay79** [comment]
FWIW I have the same issue however if I ask Siri to change the temperature she can do it fine - I just can't use the Home app to do it until after I have asked Siri.

**Comment by pedroserano** [comment]
FWIW I used to have this problem but not an issue now. I'm not sure if it was resetting iCloud Keychain and the appletv or reinstalling homebridge.

**Comment by erfinb** [comment]
I performed a restore on the Apple TV that is the hub and we are in business! Thanks all!
